### PR TITLE
[bitnami/wordpress] Improve Wordpress VIB test coverage

### PR DESCRIPTION
**Pull request by superaleks**
**Description of the change**

Increasing the test coverage of the VIB PoC on WordPress. Functional tests are written using Cypress, Infrastructure tests using Goss, and performance tests using Jmeter.  
The tests are focusing on installation correctness. They are not testing the application itself. 

**Benefits**

If we make any changes to this Helm chart, we can execute the tests and make sure we didn't break anything.

**Possible drawbacks**

**Additional information**

**Checklist**

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/vib-platform-verify.json [hunk 1]
```diff
@@ -61,8 +61,22 @@
...
+                "password": "ComplicatedPassword123!4"
+              }
+            }
+          }
+        },
+        {
+          "action_id": "jmeter",
+          "params": {
+            "config": {
+              "test_plan_file": "/.vib/wordpress/wordpress.jmx",
```
Comment (beltran-rubo) [inline .vib/vib-platform-verify.json:74]: This is the error from the verification tests:

>/.vib/wordpress/wordpress.jmx (No such file or directory)

The correct path is '.vib/wordpress/jmeter/wordpress.jmx'

File: .vib/vib-platform-verify.json [hunk 2]
```diff
@@ -61,8 +61,22 @@
...
-                "password": "S39BKWjSkh"
+                "username": "test_user",
+                "password": "ComplicatedPassword123!4"
+              }
+            }
+          }
+        },
+        {
+          "action_id": "jmeter",
+          "params": {
```
Comment (beltran-rubo) [inline .vib/vib-platform-verify.json:72]:
```suggestion
          "params": {
            "resources": {
              "path": "/.vib/wordpress/cypress"
            },
```

File: .vib/wordpress/cypress/cypress/support/commands.js [hunk 3]
```diff
@@ -1,25 +1,24 @@
...
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
```
Comment (javsalgar) [inline .vib/wordpress/cypress/cypress/support/commands.js:12]: Could you add a comment explaining this command override?

Comment (superaleks) [reply .vib/wordpress/cypress/cypress/support/commands.js:14]: Done!

**Review by beltran-rubo — APPROVED** [review]

**Pull request merged**